src/app/components/Image/index.tsx

```diff
@@ -142,8 +144,8 @@ const Image = ({
               src={src}
               width={width}
               height={height}
-              srcSet={getImgSrcSet()}
-              sizes={getImgSizes()}
+              srcSet={imgSrcSet}
+              sizes={imgSizes}
```

Comment:
This will need to be something like `{...(imgSrcSet && {srcSet: imgSrcSet})}`, same with sizes if it can be `null` too.

Reply:
damn - just changed it _to_ that! haha. Thanks. Will change back